blog/modules/ROOT/examples/sort-components/componentVersionSorter.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const VERSION_SORT_KEY = 'version-sort-key'
+
+module.exports = (what, data, sortOrder = undefined) => {
+  if (what === 'components') {
+    return sortComponents(data, sortOrder)
+  } else if (what === 'versions') {
+    return sortVersions(data)
+  } else {
+    console.log(`unexpected sort request: ${what}. Expected 'components' or 'versions`)
```

Comment:
I think we should probably throw an error otherwise it will return `undefined` and Handlebars will most likely generate a random error.